client/my-sites/plugins/marketplace/components/index.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { HorizontalRule } from '@wordpress/components';
+import styled from '@emotion/styled';
+import { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
```

Comment:
Did we mean to `import type` instead of `import`?

Reply:
TIL! will fix this

```suggestion
import type { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
```